drivers/hmis/app/models/hmis/form/definition_validator.rb

```diff
@@ -25,6 +25,12 @@ def perform(document, role = nil, skip_cded_validation: false)
     check_references(document, all_ids)
     # Check mutually exclusive attributes ("one of" on conditional objects)
     check_mutually_exclusive_attributes(document)
+
+    # Check conditions like enable_when and autofill_when.
+    # First initialize a FormDefinition to use the logic for generating an item hash, but don't persist it.
+    item_hash = Hmis::Form::Definition.new(definition: document).link_id_item_hash
```

Comment:
`link_id_item_hash` excludes Group items, which we do need to validate here since they can have conditional visibility

Reply:
Hm, yes. But we are using the `link_id_item_hash` to find the referenced question, which we only validate against if it has a `pick_list_options` or `pick_list_reference`. Group items would not have these keys anyway, so I think it is ok for them to be missing from the hash.

If we wanted to add more validation, like on EXISTS conditions, where the referenced question can also be a group, then this wouldn't work.

I guess the alternative is to make new logic locally for a link_id_item_hash that would contain all group items. Do you have any concerns that that would balloon in terms of space complexity?